### DISCUSSION TITLE
Phone number input component

### DISCUSSION THREAD

**Discussion by Flowko** [unovue/shadcn-vue#458] [Ideas]
An implementation of a Phone Input component built on top of Shadcn UI's input component.

https://shadcn-phone-input.vercel.app/

**Comment by vai** [comment]
That would be good.

**Comment by SidVerson** [comment]
Hey! i created this component, check this out!
https://shadcn-vue-phone-input.vercel.app/
https://github.com/SidVerson/shadcn-vue-phone-input.git
